Presentation/Activities/MainActivity.cs

```diff
@@ -28,17 +27,22 @@ private void LoadFragment(int id)
             switch (id)
             {
                 case Resource.Id.menu_home:
-                    fragment = new OpenWebTechnologyFragment();
+                    fragment = new OwtFragment();
                     break;
                 case Resource.Id.menu_audio:
-                    fragment = new StoriesFragment();
+                    fragment = new StoryFragment();
                     break;
                 case Resource.Id.menu_video:
                     fragment = new ContactFragment();
                     break;
             }
-            if (fragment == null) return;
-            FragmentManager.BeginTransaction().Replace(Resource.Id.content_frame, fragment).Commit();
+
```

Comment:
Make a function `DisplayFragment(Fragment fragment)` and call. It's easier.

Reply:
Done

Reply:
Done